### PR TITLE
Feature/cart list

### PR DESCRIPTION
**Pull request by VVMig**

### CODE REVIEW

File: src/app/cart-item/SelectedConfig.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  pickers,
+  route,
+}) => {
+  const pickersMap = pickers?.map((picker, index) => (
+    <Styled.SelectedPicker key={`${picker.title}-${index}`}>
+      <Styled.PickerTitle>{picker.title}:</Styled.PickerTitle>
+      {picker.items
+        ?.filter((item) => item.isSelect)
+        .map((item) => (
+          <Styled.Configuration backgroundColor={item.backgroundColor}>
```
Comment (RuslanShohanov) [inline src/app/cart-item/SelectedConfig.tsx]: Add key prop pls

File: src/modules/cart/Cart.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { CartProps } from './interfaces';
+import { Styled } from './styled';
+
+export const Cart: React.FC<CartProps> = ({ cartItems }) => {
+  return (
+    <>
+      <Header />
+      <Styled.Body>
+        {cartItems.map((item, index) => (
+          <CartItem
```
Comment (RuslanShohanov) [inline src/modules/cart/Cart.tsx:26]: Key prop?

File: src/app/cart-item/interfaces.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { CurrencyEnum } from '../buyingEnums';
+import { IPickerTypes, ISlideTypes } from '../store';
+
+export interface OperationProps {
```
Comment (RuslanShohanov) [inline src/app/cart-item/interfaces.ts:4]: You already have the same interface in cart/interfaces?

File: src/app/cart-item/interfaces.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  route: string;
+}
+
+export interface ItemInfoProps extends SelectedConfigProps, UserActionsProps {
+  title?: string;
+  manufacturer?: string;
+  description?: string;
+}
+
+export interface StyledCartItemProps extends OperationProps {
```
Comment (RuslanShohanov) [inline src/app/cart-item/interfaces.ts:25]: You already have the same interface in cart/interfaces?

File: src/app/cart-item/SelectedConfig.tsx (hunk 1 above)
Comment (VVMig) [reply src/app/cart-item/SelectedConfig.tsx]: the last commit does not have this(add breakpoints)

File: src/modules/cart/Cart.tsx (hunk 2 above)
Comment (VVMig) [reply src/modules/cart/Cart.tsx:26]: the last commit does not have this(add breakpoints)

File: src/app/cart-item/interfaces.ts (hunk 3 above)
Comment (VVMig) [reply src/app/cart-item/interfaces.ts:4]: My bad

**Review by RuslanShohanov — APPROVED** [review]

**Pull request merged**